### DISCUSSION TITLE
Zeus community vouched peers / LSP

### DISCUSSION THREAD

**Discussion by Darth-Coin** [ZeusLN/zeus#2265] [General]
As the number of Zeus users is growing, I would like to propose to build here a "vouched" list of peers or LSPs for Zeus new users.
Many new users do not know to which node they should open a new channel, apart from the default Zeus LSP Olympus.

We know that is important to have a channel with Olympus LSP but also with another 2-3-4 other good nodes / LSP.

### Important points about having a channel with Olympus LSP
[Opening an "on the fly"](https://docs.zeusln.app/for-users/embedded-node/lightning-onboarding) channel with Olympus you get the following advantages:
- excellent payment routes for your private node
- zero fees policy
- additional 100k sats inbound liquidity
- [wrapped invoices](https://docs.zeusln.app/lsp/wrapped-invoices/) for increased privacy
- route hints for better incoming payment routes
- better support in case of opening/closing channels

### Why is recommended to have multiple channels instead of just one with Olympus?
- multiple routes
- diversification of routing fees
- MPP - multi-path payment, increased privacy and liquidity
- reliability of your liquidity. If one channel is offline (peer unresponsive) you can still have alternative routes
- more balanced liquidity
- not concentrating all the liquidity "charge" on the Olympus LSP node.

Please post only those peers you really used with your Zeus LN node. Keep in mind that Zeus is using "private channels" (unannounced), not "public" routing channels. Choose wisely your peers, because not all public nodes are willing to open "private" channels. Usually "private" channels are used less than "public" channels so for a routing only node is not so "profitable" to open "private" channels, locking some funds in them.

Post here your proposal and give some details, like:

- Peer node name
- Peer link in LN explorer ([Amboss](https://amboss.space/) or [Mempool](https://mempool.space/lightning) or any other explorer you use)
- Peer fees used, if you know them
- Experience you had with it, give us some details, good or bad, aspects etc, if it was cooperative, why do you choose it, if is offering also inbound channels
- contact, if you have one or the owner is willing to be contacted
- Your personal score 1 to 10

**Comment by Darth-Coin** [comment]
### [Olympus by Zeus](https://mempool.space/lightning/node/031b301307574bbe9b9ac7b79cbe1700e31e544513eae0b5d7497483083f99e581)
- Contact: [Telegram](https://t.me/zeusLN), [Nostr](https://primal.net/p/npub1xnf02f60r9v0e5kty33a404dm79zr7z2eepyrk5gsq3m7pwvsz2sazlpr5)
- zero fee routing (for LSP channels)
- fees schedule for channels: https://docs.zeusln.app/lsp/fees
- LND node, LSP, very good routes
- you can also buy inbound channels from Olympus: https://channels.zeuslsp.com/
- is recommended to use the option of [onboarding with Lightning](https://docs.zeusln.app/for-users/embedded-node/lightning-onboarding) paying over LN for opening a new channel with Olympus. Why? Because in this way your channel will not get force closed if the balance is more on the Olympus side. If you just open a channel with funds from your onchain wallet and you keep the channel almost empty on your side, then the LSP could force close it anytime.
- score 10/10

**Comment by Darth-Coin** [comment]
### [Blixt node](https://mempool.space/lightning/node/0230a5bca558e6741460c13dd34e636da28e52afd91cf93db87ed1b0392a7466eb)
- contact: Hampus in [Blixt Telegram group](https://t.me/blixtwallet)
- 1 sat base / dynamic ppm | min 50k channel, no max.
- private channels are welcomed
- LND node based in Germany
- good node with a lot of connections to many rings and big routes.
- is a good option also to connect to other Blixt users, your payment could find a shorter route using Blixt node as hop.
- score 9/10

**Comment by Darth-Coin** [comment]
### [CoinGate](https://mempool.space/lightning/node/0242a4ae0c5bef18048fbecf995094b74bfb0f7391418d71ed394784373f41e4f3)
- contact web https://coingate.com/
- 1base / 1 ppm | not known min/max channel size | LND node based in Germany
- node with 1000+ routing channels, top 10, highly available
- private channels friendly
- used few times with 1M sats channel, no routing issues
- Coingate is a payment processor for merchants too
- score 7/10

**Comment by Darth-Coin** [comment]
### [LNbig](https://lnbig.com/#/our-nodes)
- contact: web https://lnbig.com/ | [NOSTR](https://primal.net/p/npub1ngumlqmus6xkrmvvee4yc7swh9h4uk7vpq4ddt7a2jtvkc22y0asrse3pv)
- dynamic fees and quite high | no min/max channel size | various LND nodes across the globe
- choose the right server before connecting with it, some of them have high fees
- also offering inbound channels (paid)
- LNBig nodes are very well connected with many routes
- score 7/10

**Comment by Darth-Coin** [comment]
### [Megalithic](https://mempool.space/lightning/node/038a9e56512ec98da2b5789761f7af8f280baf98a09282360cd6ff1381b5e889bf)
- contact: web [https://megalithic.me/](https://megalithic.me/) | [Telegram](https://t.me/megalith_node) | email info@megalithic.me
- min channel size: 1M sats
- offer public and private channels
- good uptime and liquidity
- [documentation page](https://docs.megalithic.me/)
- score 9/10

**Comment by Darth-Coin** [comment]
### [NERV](https://mempool.space/lightning/node/028c589131fae8c7e2103326542d568373019b50a9eb376a139a330c8545efb79a)


- contact: [Telegram](https://t.me/SynthLock), LN dev, very cooperative and knowledgeable
- min channel size: 200k sats
- offer public and private channels
- good uptime and liquidity, good fees
- score 8/10

**Comment by nikotsla** [comment]
### [NikolaTesla](https://amboss.space/node/035cf526f4c01ed693bacd7434461b9dc53d65b910fb3006110d8b42a574858fb2)

- contact: Nostr: npub1rd0z8aj4wrt7f59xfy95jtp5ue56z4vw7z2mfngnu4pjnj65vpyqxm5cc2 | email: [nikotsla@proton.me](mailto:nikotsla@proton.me) | web [nikotsla](https://nikotsla.github.io/)
- min channel size: 100k sats
- offer public
- offer private: Closed policy 2 months without traffic  
- LND, dynamic fees (0-300), min/max HTLC, peerswap enabled
- Long time noderunner, easy to reach
- score 5/10 (Tor Only)

**Comment by Darth-Coin** [comment]
### [Coincept](https://amboss.space/node/0202e7727ee46d943a44c7bf0dce14220a839630dee67d03446bf935178b55a0df)
- contact: web https://coincept.com/ | [Telegram](https://t.me/CoinceptLN) | Email: coincept.lightning@proton.me
- min channel size: 1M sats
- option for private and public channels
- option to pay with onchain or LN invoice
- well known node runner
- good uptime, good liquidity, hybrid node
- score 9/10
